### ISSUE TITLE
solving 1D heat transfer without heat generation

### ISSUE THREAD

**Issue by mrtolzhas** [lululxvi/deepxde#532]
Dear Lulu, thanks for your work.

I am trying to solve very simple second order differential equation for 1D heat steady-state heat transfer without heat generation, where

dy_xx = 0,

When dy_xx is equal to 0, NN cannot handle it. 
"""Backend supported: tensorflow.compat.v1, tensorflow, pytorch"""
import deepxde as dde
import matplotlib.pyplot as plt
import numpy as np
# Import tf if using backend tensorflow.compat.v1 or tensorflow
from deepxde.backend import tf
# Import torch if using backend pytorch
# import torch

def pde(x, y):
    dy_xx = dde.grad.hessian(y, x)
   
    return  dy_xx
   

def func(x):
    return 100+(-50)*(x)

def boundary(x, _):
    return np.isclose(x[0],0) or np.isclose(x[0],1)

#def boundary(x, on_boundary):
#    return on_boundary

geom = dde.geometry.Interval(0,1)
bc = dde.DirichletBC(geom, func, boundary)
data = dde.data.PDE(geom, pde, bc, 16, 2, solution=func, num_test=100)

layer_size = [1] + [50] * 3 + [1]
activation = "tanh"
initializer = "Glorot uniform"
net = dde.maps.FNN(layer_size, activation, initializer)

model = dde.Model(data, net)
model.compile("adam", lr=0.001, metrics=["l2 relative error"])

losshistory, train_state = model.train(epochs=20000)
# Optional: Save the model during training.
# checkpointer = dde.callbacks.ModelCheckpoint(
#     "model/model", verbose=1, save_better_only=True
# )
# Optional: Save the movie of the network solution during training.
# ImageMagick (https://imagemagick.org/) is required to generate the movie.
# movie = dde.callbacks.MovieDumper(
#     "model/movie", [-1], [1], period=100, save_spectrum=True, y_reference=func
# )
# losshistory, train_state = model.train(epochs=10000, callbacks=[checkpointer, movie])

dde.saveplot(losshistory, train_state, issave=True, isplot=True)

# Optional: Restore the saved model with the smallest training loss
# model.restore(f"model/model-{train_state.best_step}.ckpt", verbose=1)
# Plot PDE residual
x = geom.uniform_points(1000, True)
y = model.predict(x, operator=pde)
plt.figure()
plt.plot(x, y)
plt.xlabel("x")
plt.ylabel("PDE residual")
plt.show()

![123123](https://user-images.githubusercontent.com/85702994/155071608-e02ce8ba-39ab-412e-9524-775660dc6539.png)

but if we add any coefficient, there is no problem. 

def pde(x, y):
    dy_xx = dde.grad.hessian(y, x)
   
    return  5*dy_xx

![image](https://user-images.githubusercontent.com/85702994/155071761-a3e003f1-8f9b-4aad-9e84-88a93cf23784.png)

Can it handle second order equations which equal to 0?

**Comment by lululxvi** [comment]
See FAQ: Q: I failed to train the network or get the right solution, e.g., large training loss, unbalanced losses.
